### PR TITLE
plugins: Fix PitchShift windowSize freeze

### LINKED ISSUE #3183: PitchShift can crash the server
Accidentally did something equivalent to this today:

```{ PitchShift.ar(SinOsc.ar, SinOsc.ar) }.play```

I know window size is not modulatable, but this shouldn't bring the server down.

This behaves similarly: ```{ PitchShift.ar(SinOsc.ar, 0) }.play ```

### PR DESCRIPTION
**Pull request by snappizz**
fixes #3183 by setting minimum windowsize to 3 sample periods

### CODE REVIEW

File: server/plugins/DelayUGens.cpp [hunk 1]
```diff
@@ -5681,6 +5681,14 @@ void PitchShift_Ctor(PitchShift *unit)
 	pchratio = ZIN0(2);
 	winsize = ZIN0(1);
 
+	// If the window size is <= 2 samples, scsynth will freeze (presumably some
```
Comment (brianlheim) [inline server/plugins/DelayUGens.cpp]: You should add a FIXME / TODO to this, since the original problem is still not fully understood

Comment (brianlheim) [reply server/plugins/DelayUGens.cpp]: I tested this and it works fine; please just add a FIXME and I'm happy to merge! I'm also going to add a note about fixmes/todos to the style guide

**Comment by brianlheim** [comment]
oh also, before I forget - probably should add a unit test

**Comment by brianlheim** [comment]
ping

**Comment by snappizz** [comment]
sorry, kinda busy now -- i'll get on this next time i have time to work on SC

**Pull request merged**